subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/local/LocalReservationClient.kt

```diff
@@ -46,7 +46,7 @@ class LocalReservationClient(
 
         val devicesChannel: Channel<WorkerDevice> = reservations
             .iterateInParallel { _, reservation ->
-                val deploymentName = generateDeploymentName()
+                val deploymentName = "local-stub"
```

Comment:
Why do we name it deploymentName?

Reply:
It's a fake. I'll delete it after refactoring the `ReservationClient` interface. 
It doesn't fit local runs